nimterop/getters.nim

```diff
@@ -306,4 +321,26 @@ proc getNimExpression*(expr: string): string =
 
 proc getSplitComma*(joined: seq[string]): seq[string] =
   for i in joined:
-    result = result.concat(i.split(","))
\ No newline at end of file
+    result = result.concat(i.split(","))
+
+proc dll*(path: string): string =
+  let
+    (dir, name, _) = path.splitFile()
+
+  result = dir / (DynlibFormat % name)
+
+proc loadPlugin*(fullpath: string) =
```

Comment:
`fullpath` is not self-documenting, not clear whether it refers to foo.nim or libfoo.dylib; 
maybe: sourcePath ?